doc/api/fs.md

```diff
@@ -3605,10 +3605,10 @@ recommended.
 1. Any specified file descriptor has to support writing.
 2. If a file descriptor is specified as the `file`, it will not be closed
 automatically.
-3. The writing will begin at the beginning of the file. For example, if the
-file already had `'Hello World'` and the newly written content is `'Aloha'`,
-then the contents of the file would be `'Aloha World'`, rather than just
-`'Aloha'`.
+3. The writing will begin at the current position. For example, if the string
+`'Hello'` is written to the file descriptor, and if `'World'` is written with
```

Comment:
I would suggest a string of `', World'`. Or perhaps, the first string could be `'Dogs\n'`, and the second `'Cats\n'`? The docs as written generate a misspelled phrase, which to me suggests its demonstrating a side-effect of some kind of misuse of the API, rather than a working-as-designed-and-useful capability.

Reply:
I included your suggestion, the `, World`. I didn't change it to Dogs and Cats because, `readFile` also uses Hello World as the example. Perhaps we can change them together in a separate PR. What do you think?